core/src/test/java/io/questdb/cutlass/pgwire/PGSecurityTest.java

```diff
@@ -0,0 +1,216 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package io.questdb.cutlass.pgwire;
+
+import io.questdb.std.datetime.microtime.TimestampFormatCompiler;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.postgresql.util.PSQLException;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+
+import static io.questdb.test.tools.TestUtils.assertContains;
+import static org.junit.Assert.fail;
+
+public class PGSecurityTest extends BasePGTest {
+
+    @ClassRule
+    public static TemporaryFolder backup = new TemporaryFolder();
+
+    private static final PGWireConfiguration READ_ONLY_CONF = new DefaultPGWireConfiguration() {
+        @Override
+        public boolean readOnlySecurityContext() {
+            return true;
+        }
+    };
+
```

Comment:
Please consider either : 
- extracting these two lines to AbstractCairoTest (without @BeforeClass annotation) and call it from PGSecurityTest.configureForBackups() or 
- moving changes from AbstractCairoTest to this class and overriding configuration in @BeforeClass method .

Reply:
👍